Source/WebKit/UIProcess/API/glib/WebKitNotification.cpp

```diff
@@ -51,12 +51,11 @@ enum {
 };
 
 struct _WebKitNotificationPrivate {
+    RefPtr<WebKit::WebNotification> notification;
```

Comment:
Surely this should be:

```suggestion
    Ref<WebKit::WebNotification> notification;
```

because you're treating it as if it is not nullable everywhere.

Reply:
That requires a value provided in the constructor and that's not possible with GObject priv struct.